### PR TITLE
update standalone deployment manifests

### PR DESCRIPTION
**Pull request by vitus133**
This PR fixes manifests for daemon standalone deployment.
- Switch from cluster role to namespaced role
- Label the namespace for pod-security compliance
In addition adds development artifacts to .gitignore.
All together makes standalone deployment compatible with the repository README.md
/assign @aneeshkp 
/cc @josephdrichard

### CODE REVIEW

File: deploy/linuxptp-daemon.yaml [hunk 1]
```diff
@@ -26,11 +26,15 @@ spec:
       - name: linuxptp-daemon-container
         securityContext:
           privileged: true
-        image: openshift.io/linuxptp-daemon
+        image: quay.io/vgrinber/linuxptp-daemon:2
```
Comment (aneeshkp) [inline deploy/linuxptp-daemon.yaml:29]: we can't merge this. What is this required for ?

File: deploy/linuxptp-daemon.yaml [hunk 2]
```diff
@@ -26,11 +26,15 @@ spec:
...
         command: [ "/bin/bash", "-c", "--" ]
         args: [ "/usr/local/bin/ptp --alsologtostderr" ]
-        imagePullPolicy: IfNotPresent
+        # command: [ "/bin/sleep" ]
+        # args: [ "inf" ]
+        imagePullPolicy: Always
```
Comment (aneeshkp) [inline deploy/linuxptp-daemon.yaml:34]: Shouldn't be Always

**Comment by aneeshkp** [comment]
I am not sure we support standalone deployment any more. Is there any need for it ?

**Comment by vitus133** [comment]
Sorry this is a mistake.

**Pull request closed without merging**